fastlane/lib/fastlane/module.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../fastlane_core/lib/fastlane_core/require_relative_helper'
```

Comment:
check if this is the current state of the module stuff that got removed from fastlaner.rb